Makefile

```diff
@@ -25,14 +24,19 @@ TOOLS_DIR := hack/tools
 -include $(HACK_DIRECTORY)/tools.mk
 include hack/tools.mk
 
-.PHONY: start
-start:
+GOIMPORTSREVISER_VERSION := v3.5.6
```

Comment:
Is this used?

Reply:
Yes, this is used in the included `tools.mk` files to download a version-pinned `goimports-reviser` binary.